node/runtime/pangoro/src/pangolin_messages.rs

```diff
@@ -28,12 +28,12 @@ use sp_runtime::{traits::Zero, FixedPointNumber, FixedU128};
 use sp_std::{convert::TryFrom, ops::RangeInclusive};
 // --- darwinia-network ---
 use crate::*;
-use bridge_primitives::{
+pub use darwinia_balances::{Instance1 as RingInstance, Instance2 as KtonInstance};
+use dp_s2s::{CallParams, CreatePayload};
+use drml_bridge_primitives::{
```

Comment:
Why not `dp_bridge`? Because we already have some crates named `dp_s2s`.

Reply:
To avoid duplicate name.